packages/react-magma-dom/src/components/Accordion/useAccordionItem.ts

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+
+import { AccordionContext } from './useAccordion';
+import { useGenerateId } from '../../utils';
+import { isArray } from 'lodash';
+
+export interface UseAccordionItemProps {
+  /**
+   * @internal
+   */
+  index?: number;
+  isDisabled?: boolean;
+  testId?: string;
+}
+
+interface AccordionItemContextInterface {
+  buttonId?: string;
+  index?: number;
+  isDisabled?: boolean;
+  isExpanded?: boolean;
+  panelId?: string;
+  setIsExpanded?: any;
```

Comment:
there is a type for state updates. 
```suggestion
  setIsExpanded?: React.Dispatch<React.SetStateAction<boolean>>;
```